### PR TITLE
fix/login-button-issue

### PR DESCRIPTION
**Pull request by athul-rs**
## What

- Issue resolved where multiple login buttons where displayed.
- Redesigned Login page.

## Why

-

## How

- Updated changes in HTML and CSS.

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Screenshots

image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: frontend/src/components/log-in/Login.jsx [hunk 1]
```diff
@@ -37,12 +37,6 @@ function Login() {
               >
                 Login
               </Button>
-              <Button
-                className="signin-button button-margin"
-                onClick={handleSignin}
```
Comment (tahierhussain) [inline frontend/src/components/log-in/Login.jsx:42]: @athul-rs Can you remove `handleSignin` function as well?

Comment (athul-rs) [reply frontend/src/components/log-in/Login.jsx:42]: @tahierhussain updated

**Review by tahierhussain — APPROVED** [review]
LGTM

**Review by mohamed-siddhiq — APPROVED** [review]
Looks good

**Pull request merged**